torchopenl3/utils.py

```diff
@@ -4,14 +4,17 @@
 TARGET_SR = 48000
 
 
+TARGET_SR = 48000
+
+
 def center_audio(audio, frame_len):
     return torch.nn.functional.pad(
         audio, (int(frame_len / 2.0), 0), mode="constant", value=0
     )
 
 
 def pad_audio(audio, frame_len, hop_len):
-    audio_len = audio.size()[0]
+    audio_len = audio.size()[1]
```

Comment:
Does this work with both naudio x nsamples x nchannels AND naudio x nsamples ?

Can you write a test to prove it works right in both cases, using random white noise [-1, 1]?